app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file the MyExperience screen for the STA Volunteer App
+ */
+import {
+  Heading,
+  VStack,
+  HStack,
+  Checkbox,
+  ScrollView,
+  Flex,
+  Button,
+  Box,
+} from 'native-base'
+import StaRibbon from '../Components/StaRibbon'
+import StaThemeLogo from '@/NativeBase/Assets/Images/Logos/sta-ribbon-logo.svg'
+import FreeSearchBar from '../Components/FreeSearchBar'
+import React from 'react'
+import { RoleGroup, roleGroups } from '@/Services/modules/projects/roleGroups'
+
+const MyExperienceContainer = () => {
+  const [skillsValue, setSkillsValue] = React.useState<string[]>([])
+  const [searchTxt, setSearchTxt] = React.useState<string>('') // search text
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) => {
+    return roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase())
+  })
+
+  return (
+    <>
+      <HStack marginTop={-10} marginBottom={12} alignItems={'flex-end'}>
+        <Heading size="md" fontWeight="bold" marginRight={-5}>
+          My Experience
+        </Heading>
+        <Flex flex={1} alignItems={'flex-end'}>
+          <StaThemeLogo />
```

Comment:
The logo seems longer than the design, I wonder if you could push it up a bit?  One way to do that could be to limit the height and hide the overflow.   

Also, on my screen (iPhone) there appears to be a little too much white space under the title.

Reply:
@PammyLo I have reduce the whitespace between the Heading and searchbar. As for the logo it looks fine on my mobiles but when i tried what you suggested the image becomes skewed.

Reply:
Hi David, it's looking really great. Just on my phone I can't see the last two colours, yellow and red, and recall Joanna saying all 7 should be visible. It's a CSS challenge for sure..
The whitespace above the search bar is now fine but there still seems to be a lot between title and STA ribbon.

Reply:
Hi Pamela that's unfortunate. Iam not having a lot with this ticket. I suspect it's down to the screen size you are using and as is why you can't view all the colours of the logo. I am not honestly sure how I can resolve this issue as each time I change  something to the logo it causes another issue in sizing.